packages/expo-ui/android/src/main/java/expo/modules/ui/SwitchView.kt

```diff
@@ -12,20 +12,20 @@ import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.views.ComposeProps
 
 data class SwitchProps(
-  val checked: MutableState<Boolean> = mutableStateOf(false),
+  val value: MutableState<Boolean> = mutableStateOf(false),
   val variant: MutableState<String> = mutableStateOf("switch")
 ) : ComposeProps
 
 class SwitchView(context: Context, appContext: AppContext) : ExpoComposeView<SwitchProps>(context, appContext) {
   override val props = SwitchProps()
-  private val onCheckedChanged by EventDispatcher()
+  private val onValueChange by EventDispatcher()
 
   init {
     setContent {
-      val (checked) = props.checked
+      val (checked) = props.value
       val (variant) = props.variant
       val onCheckedChange = { checked: Boolean ->
-        onCheckedChanged(mapOf("checked" to checked))
+        onValueChange(mapOf("value" to checked))
```

Comment:
Not sure if I understand this one, should it be value or checked here? :)

Reply:
The `onValueChange` is the JS callback, `onCheckedChange` is the native callback for the compose component. The native argument is called `onCheckedChange`, so I named the variable that we pass to it the same.